examples/Forms/MultiLineTextInput.md

```diff
@@ -0,0 +1,149 @@
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  name="course-notes"
+  label="Course notes"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with an error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  name="course-notes"
+  label="Course notes"
+  placeholder="Your text here"
+  errorMessage="Please provide a value for this field"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden label
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  name="course-notes"
+  label="Course notes"
+  placeholder="Your text here"
+  isLabelVisible={false}
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Required multi-line text box
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  name="course-notes"
+  label="Course notes"
+  placeholder="Your text here"
+  isRequired
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Disabled multi-line text box
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('Some text that you can\'t change');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  name="course-notes"
+  label="Course notes"
+  placeholder="Your text here"
+  isDisabled
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box using `useRef` to focus the element programatically
+```jsx
+import { MultiLineTextInput, Button, VARIANT } from 'mark-one';
+import { useState, useRef } from 'react';
+
+const [value, setValue] = useState('');
+
+const ref = useRef(null);
+const onButtonClick = () => {
+  if (ref.current) {
+    ref.current.focus();
+  }
+};
+<>
+  <Button
+    id="testButton"
+    onClick={onButtonClick}
+    variant={VARIANT.INFO}
+  >
+    Focus the input
+  </Button>
+  <MultiLineTextInput
+    id="text input"
+    name="course-notes"
+    label="Course Notes"
+    placeholder="Your text here"
+    onChange={() => {}}
+    forwardRef={ref}
+  />
+</>
+```
+
+
+
+Multi-line text box with a `placeholder` set
+```jsx
+import { MultiLineTextInput, Button, VARIANT } from 'mark-one';
+import { useState, useRef } from 'react';
+
+const [value, setValue] = useState('');
+  <MultiLineTextInput
+    id="text input"
+    name="course-notes"
+    label="Course Notes"
+    placeholder="Your text here"
+    onChange={() => {}}
+    forwardRef={ref}
+  />
```

Comment:
We could either remove or define the ref for this example:
![image](https://user-images.githubusercontent.com/29589689/152230675-6aee5058-e388-41bc-90c9-10e90ca48613.png)

Reply:
D'oh! Probably makes sense to remove it. Fixed that in 96e6a04